### PR TITLE
nRF24 small improvements.

### PR DESCRIPTION
**Pull request by Guglio95**
Hi Jan, 

I'm using your nRF24 driver to scan over frequencies, I made some small changes:

- `isCarrierDetected()`, to detect whatever the carrier is detected.
- I'm attaching notes on modified lines.

Please, let me know.

### CODE REVIEW

File: src/modules/nRF24/nRF24.cpp [hunk 1]
```diff
@@ -230,14 +230,9 @@ int16_t nRF24::setFrequency(int16_t freq) {
     return(ERR_INVALID_FREQUENCY);
   }
 
-  // set mode to standby
-  int16_t state = standby();
```
Comment (Guglio95) [inline src/modules/nRF24/nRF24.cpp]: Seems that there's no need to put modem in standby to change frequency; 
I've tested it and double checked on other nRF24 drivers.

File: src/modules/nRF24/nRF24.cpp [hunk 2]
```diff
@@ -438,6 +438,13 @@ size_t nRF24::getPacketLength(bool update) {
 }
 
 int16_t nRF24::setCrcFiltering(bool crcOn) {
+  // Auto Ack needs to be disabled in order to disable CRC.
```
Comment (Guglio95) [inline src/modules/nRF24/nRF24.cpp]: Enable CRC is forced high if one of the bits in EN_AA is high (datasheet, page 54)

**Comment by jgromes** [comment]
Looks like there's an ambiguity in `setAutoAck(false)` on Arduino Due: https://travis-ci.org/jgromes/RadioLib/jobs/655399163

**Comment by jgromes** [comment]
Checks out now, thanks! I didn't want to change it in master since that would cause a conflict.

**Pull request merged**